indigo/distribution.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 # ROS distribution file
-# see REP 143: http://ros.org/reps/rep-0143.html
+# see REP 141: http://ros.org/reps/rep-0141.html
```

Comment:
We changed the file and need to get a new version of rosdistro released which will generated the same content. That's why we're holding for a sync.